modules/multi-arch-deleting-podplacment-config-using-web-console.adoc

```diff
@@ -7,7 +7,9 @@
 
 = Deleting the ClusterPodPlacementConfig object by using the web console
 
-You can delete the `ClusterPodPlacementConfig` object by using the {product-title} web console.
+You can create only one instance of the `ClusterPodPlacementConfig` object. If you want to recreate this object, you must first delete the existing instance.
```

Comment:
```suggestion
You can create only one instance of the `ClusterPodPlacementConfig` object. If you want to re-create this object, you must first delete the existing instance.
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#r

Reply:
Updated as per the suggestion.